src/environments/environment.prod.ts

```diff
@@ -2,5 +2,5 @@ export const environment = {
   production: true,
   test: false,
   // Change this to Docker address
-  couchAddress: window.location.hostname + ':planet-db-port/'
+  couchAddress: window.location.protocol + '//' + window.location.hostname + ':planet-db-port/'
```

Comment:
@mappuji think we need to keep changing this line ... you remove this later

Reply:
We change it dynamically via shell script @dogi